### ISSUE TITLE
Accessibility: Color contrast is insufficient

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#433]
**Description**
Color contrast for all elements on a page should be a 4.5 to 1 ratio for body text and 3 to 1 for large heading text. It's better to default to 4.5 to 1 for all of these just to be safe. There are a number of areas that have this issue. All areas are indicated in the Solution section along with how to fix it.

[This report](https://my2.siteimprove.com/Inspector/602695/Accessibility/Page?pageId=40341591380&impmd=3CC33A985F387DC37EA87D3D5A2306C4#/Criterion/1.4.3/Check/19) shows all of the issues that are present on the majority of pages across the site.

**Solution**
1. Exhibit Title and Subtitle - Background for this section should change to black for ease of readability if content owners have not added a banner image. In addition, content owners must be aware of color contrast used with regards to the banner image they use.
2. Navigation bar - All of the navigation items are throwing errors with this. Would recommend making the background color for the navigation bar a solid color (not using opacity). This could be black to match the background for exhibit title/subtitle or another that would allow the text to be 4.5 to 1 ratio.
3. Breadcrumbs - The gray text of the active page is too light. Would recommend making all text black or a dark gray for this type of content.
4. Catalog items - All headings for catalog items (Description, Cartographer, Date, etc.) have the same issue as the breadcrumbs above and would recommend matching this color to the one used there.
5. Footer - The Cornell Exhibits and Spotlights versions text is too light. Would recommend using the same color as breadcrumbs and catalog items headings for consistency.

**Comment by mhk33** [comment]
https://my2.siteimprove.com/Accessibility/1081272/NextGen/Issue?ruleId=69&pageSegments=&issueKind=1&conformance=&siteGoal=

**Comment by chrisrlc** [comment]
I just see 1 page with this issue now in SiteImprove:
https://my2.siteimprove.com/Accessibility/1040071/NextGen/Issue/1?ruleId=69&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteGoal=0,1

Looks like the Item Slideshow widget has color contrast issue with the white captions on the blue background.
